### DISCUSSION TITLE
End-of-Support of v2

### DISCUSSION THREAD

**Discussion by klarcheveque** [aws/aws-sdk-js#4503] [Q&A]
Following this message in README.MD 

> We are formalizing our plans to make the Maintenance Announcement (Phase 2) for AWS SDK for JavaScript v2 in 2023. Please refer to the [AWS SDKs and Tools maintenance policy](https://docs.aws.amazon.com/sdkref/latest/guide/maint-policy.html) for further details.

and this documentation https://docs.aws.amazon.com/sdkref/latest/guide/maint-policy.html

For roadmap planning as a user of aws-sdk-js V2, It would mean that aws-sdk-js V2 won't be End-of-Support before April 2025?

This discussion don't answer my question https://github.com/aws/aws-sdk-js/discussions/4190

**Comment by RanVaknin** [comment]
Hi @klarcheveque,

This is not the Maintenance announcement itself. This is a notice to encourage customers to move to v3.  The SDK is still ironing out some final details before officially moving to maintenance mode. 

Thanks,
Ran~
